components/atom/switch/src/index.scss

```diff
@@ -0,0 +1,121 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$h-atom-switch--default: $sz-icon-l !default;
+$w-atom-switch--default: 34px !default;
+$h-atom-switch-container: 22px !default;
+
+$h-atom-switch--large: $sz-icon-xl !default;
+$w-atom-switch--large: 49px !default;
+$h-atom-switch-container---large: 32px !default;
+
+$c-AtomSwitch-active: color-variation($c-primary, 1) !default;
+$c-AtomSwitch-default: color-variation($c-gray, 3) !default;
+
+.sui-AtomSwitch {
+  cursor: pointer;
+  display: inline-block;
+
+  &-toggleType {
+    .sui-AtomSwitch--container {
+      margin-top: $p-l;
+    }
+  }
+
+  &--singleType {
+    .sui-AtomSwitch--container .sui-AtomSwitch--inputContainer {
+      margin-left: $m-l;
+    }
+  }
+
+  &--container {
+    display: flex;
+    height: $h-atom-switch--default;
+    justify-content: center;
+    padding-bottom: $p-s;
+    padding-top: $p-s;
+  }
+
+  &--text {
+    display: flex;
+    flex-direction: column;
+    font-size: $fz-m;
+    justify-content: center;
+
+    &.sui-AtomSwitch--right {
+      padding-left: $p-m;
+    }
+
+    &.sui-AtomSwitch--left {
+      padding-right: $p-m;
+    }
+  }
+
+  .sui-AtomSwitch--inputContainer {
+    background-color: $c-AtomSwitch-default;
+    border-radius: $w-atom-switch--default / 2;
+    border: solid 1px color-variation($c-AtomSwitch-default, $c-dark-step);
+    cursor: pointer;
+    height: $h-atom-switch-container;
+    transition: background-color, border 0.4s ease-in-out;
+    width: $w-atom-switch--default;
+
+    .sui-AtomSwitch--circle {
+      background-color: $c-white;
+      border-radius: ($h-atom-switch-container - 2) / 2;
+      border: solid 1px $c-white;
+      box-shadow: 2px 0 2px 0 $c-gray-light;
+      height: $h-atom-switch-container - 2;
+      transition: margin 0.4s ease-in-out;
+      width: $h-atom-switch-container - 2;
+
+      &.sui-AtomSwitch--toggle {
+        margin-left: 12px;
```

Comment:
please use margin var

Reply:
I created some new vars, as this margin is used to place the circle within the component